src/Components/Container.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+import React from "react";
+import withStyles from "elevate-ui/withStyles";
+import classNames from "classnames";
+
+type Props = {
+  className?: string,
+  classes: Object,
+  children: any,
+  paddingTop?: number,
+  paddingRight?: number,
+  paddingBottom?: number,
+  paddingLeft?: number,
+  style?: Object,
+};
+
+const Container = ({ className, classes, children, style }: Props) => (
+  <div className={classNames(classes.root, className)} style={style}>
+    {children}
+  </div>
+);
+
+const styles = (props) => ({
```

Comment:
This function should take in `theme` instead of `props`. Since we're not using either, we can probably drop it.